src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -141,7 +141,7 @@ internal class RazorLanguageServerClient : ILanguageClient, ILanguageClientCusto
             // Initialize Logging Infrastructure
             _loggerProvider = (LogHubLoggerProvider)await _logHubLoggerProviderFactory.GetOrCreateAsync(LogFileIdentifier, token).ConfigureAwait(false);
 
-            _server = await RazorLanguageServerWrapper.CreateAsync(serverStream, serverStream, traceLevel, _projectSnapshotManagerDispatcher, ConfigureLanguageServer, _languageServerFeatureOptions).ConfigureAwait(false);
+            _server = await RazorLanguageServerWrapper.Create(serverStream, serverStream, traceLevel, _projectSnapshotManagerDispatcher, ConfigureLanguageServer, _languageServerFeatureOptions).ConfigureAwait(false);
```

Comment:
Looks like this is a legit build error that should probably be addressed in this PR.

Reply:
Good catch. In my effort to avoid duplicating changes while sending PR's simultaneously I missed that one.